assets/app/views/_pod-template.html

```diff
@@ -3,7 +3,19 @@
   podTemplate
   imagesByDockerReference (optional)
   builds (optional)
+  detailed (optional)
+  addHealthCheckUrl (optional)
  -->
+
+ <!-- Prompt to add health checks if missing. -->
+<div ng-if="detailed && addHealthCheckUrl && !(podTemplate | hasHealthChecks)" class="alert alert-info">
+  <span class="pficon pficon-info" aria-hidden="true"></span>
+  <span ng-if="podTemplate.spec.containers.length === 1">This container has no health checks.</span>
+  <span ng-if="podTemplate.spec.containers.length > 1">Not all containers have health checks.</span>
+  Health checks ensure your application is running correctly.
```

Comment:
health check is repeated like three times, feels like its getting redundant, maybe just:
This container has no health checks, which ensure your application is running correctly.

or does that read too weird...